execution_engine/src/core/engine_state/mod.rs

```diff
@@ -829,6 +832,19 @@ where
                 .write(locked_funds_period_key, value);
         }
 
+        if let Some(new_unbonding_delay) = upgrade_config.new_unbonding_delay() {
```

Comment:
I'd like to try to keep the `auction_contract` named keys together, so move this up.